packages/rollup/src/index.ts

```diff
@@ -71,10 +68,13 @@ export function imagetools(userOptions: Partial<RollupPluginOptions> = {}): Plug
 
       for (const config of imageConfigs) {
         const { transforms, warnings } = generateTransforms(config, transformFactories)
-        warnings.forEach((warning) => this.warn(warning))
```

Comment:
Logging the warnings has to happen after `applyTransforms`, because this is where they are triggered.